src/NuGet.Core/NuGet.Configuration/PackageSourceMapping/PackageSourceMappingConfiguration.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace NuGet.Configuration
 {
-    public class PackageNamespacesConfiguration
+    public class PackageSourceMappingConfiguration
```

Comment:
naming: Should this simply be `PackageSourceMapping` ? 

Configuration might be superfluous? 
Hoping @zivkan can provide some feedback as well.

Reply:
`PackageSourceMappingConfiguration` is too long, so above suggestion is good for me. If we do above then need to do same for `PackageSourceMappingConfigurationUtility` and other variables too.

Reply:
Yes, I like `PackageSourceMapping`. Pretending that I'm working on something in restore, unrelated to the package source mapping feature, I feel like it should be sufficiently obvious it's a class that holds a mapping between package prefixes are package sources. Therefore, I don't think the name `Configuration` adds clarification. I also can't think why there'd be any `PackageSourceMapping` class in any assembly other than `NuGet.Configuration`, so I think it passes all the naming tests I can think of.

Reply:
I renamed the type to `PackageSourceMapping` in the latest commit which already passed all CI checks.